client/state/utils.js

```diff
@@ -361,6 +347,41 @@ export const withSchemaValidation = ( schema, reducer ) => {
 	return wrappedReducer;
 };
 
+/**
+ * Wraps a reducer such that it won't persist any state to the browser's local storage
+ *
+ * @example prevent a simple reducer from persisting
+ * const age = ( state = 0, { type } ) =>
+ *   GROW === type
+ *     ? state + 1
+ *     : state
+ *
+ * export default combineReducers( {
+ *   age: withoutPersistence( age )
+ * } )
+ *
+ * @example preventing a large reducer from persisting
+ * const posts = withoutPersistence( keyedReducer( 'postId', post ) )
+ *
+ * @param {Function} reducer original reducer
+ * @returns {Function} wrapped reducer
+ */
+export const withoutPersistence = reducer => {
```

Comment:
We have one usage in `client/state/automated-transfer/reducer.js` should we allow usage outside of the util, or remove it? Mainly I'd want to avoid any confusion of how not to persist. (eg do nothing).

Reply:
In the AT reducer, the usage of `withoutPersistence` attempts to prevent persisting the `fetchingStatus` property of an otherwise persisted state. But I'm afraid the code is buggy.

Neither of the two other reducers passed to `combineReducers( { eligibility, status, fetchingStatus } )` has a schema or `hasCustomPersistence`. Therefore, they will be automatically wrapped in `withoutPersistence` and nothing will be persisted.

To fix the bug and persist the `eligibility` and `status` properties, their reducers need to get the `hasCustomPersistence` flag. The `fetchingStatus` doesn't need to be wrapped in `withoutPersistence` -- `combineReducers` will do that.

Conclusion: `withoutPersistence` can be removed from there and it ends up with zero usages.

I'd prefer to do the work described above in another PR.